ava-labs/core-mobile
# Fix double notification on Android

## Description

**Ticket: [CP-9567]** 

This fixes double notification on Android devices when app is in **background** since FirebaseSDK automatically handles FCM with `notification` payload.

## Testing
- make sure that notifications are enabled in system and app settings
- make sure you are receiving any notifications and hear sound 
- use wallet on another device to send some funds on testing device (can be same wallet)
- observe that no double sound is heard or double notifications present

## Checklist

Please check all that apply (if applicable)


[CP-9567]: https://ava-labs.atlassian.net/browse/CP-9567?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
